### PR TITLE
Improve the API docs, module MD, and package MD

### PR DESCRIPTION
**Pull request by BuddhiWathsala**
## Purpose
$titile

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: time-ballerina/Module.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+## Module Overview
```
Comment (daneshk) [inline time-ballerina/Module.md:1]:
```suggestion
## Overview
```

File: time-ballerina/Package.md [hunk 2]
```diff
@@ -1,73 +1,18 @@
...
-### UTC Time
-The `time:Utc` is the tuple representation of the UTC. The UTC represents the number of seconds from a
-specified epoch. Here, the epoch is the UNIX epoch of 1970-01-01T00:00:00Z.
+This package provides implementations related to UTC and localized time generation and conversions.
 
-Use the following API to get the current epoch time:
-```ballerina
-time:Utc utc = time:utcNow();
-```
+For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/learn/by-example/).
```
Comment (daneshk) [inline time-ballerina/Package.md:8]:
```suggestion
```

File: time-ballerina/Package.md [hunk 3]
```diff
@@ -1,73 +1,16 @@
...
-// Converts a given Civil to an UTC
-time:Utc utc = time:utcFromCivil(civil);
-```
-
-To learn more about these APIs, see the [time example](https://ballerina.io/learn/by-example/time.html).
+### Useful Links
 
+- Discuss the code changes of the Ballerina project in ballerina-dev@googlegroups.com.
+- Chat live with us via our [Slack channel](https://ballerina.io/community/slack/).
+- Post all technical questions on Stack Overflow with the #ballerina tag.
```
Comment (MadhukaHarith92) [inline time-ballerina/Package.md:16]: Add link here.

File: time-ballerina/Package.md [hunk 4]
```diff
@@ -1,73 +1,16 @@
...
-time:Civil civil = time:utcToCivil(utc);
-
-// Converts a given Civil to an UTC
-time:Utc utc = time:utcFromCivil(civil);
-```
-
-To learn more about these APIs, see the [time example](https://ballerina.io/learn/by-example/time.html).
+### Useful Links
 
+- Discuss the code changes of the Ballerina project in ballerina-dev@googlegroups.com.
```
Comment (MadhukaHarith92) [inline time-ballerina/Package.md:14]: Add link here.

File: time-ballerina/Module.md (hunk 1 above)
Comment (BuddhiWathsala) [reply time-ballerina/Module.md]: Fixed in here https://github.com/ballerina-platform/module-ballerina-time/pull/139/commits/cdbfcf0e115ca2df505da49affbb976599680475

File: time-ballerina/Package.md (hunk 2 above)
Comment (BuddhiWathsala) [reply time-ballerina/Package.md]: Fixed in here https://github.com/ballerina-platform/module-ballerina-time/pull/139/commits/b4c40348f400b1cc84a410b543caa312def5102d

File: time-ballerina/Package.md (hunk 3 above)
Comment (BuddhiWathsala) [reply time-ballerina/Package.md]: Fixed in here https://github.com/ballerina-platform/module-ballerina-time/pull/139/commits/864b73f9a6321c689a098d542e27f049f608c2e4

File: time-ballerina/Package.md (hunk 4 above)
Comment (BuddhiWathsala) [reply time-ballerina/Package.md]: Fixed in here https://github.com/ballerina-platform/module-ballerina-time/pull/139/commits/864b73f9a6321c689a098d542e27f049f608c2e4

**Review by MadhukaHarith92 — APPROVED** [review]
LGTM

File: time-ballerina/Module.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
+## Overview
+
+This module provides a set of APIs that have the capabilities to generate and manipulate UTC and localized time.
+
+In cloud computing, the most essential type of time is UTC. Coordinated Universal Time (UTC) is the primary time standard on which the world agreed. 
+UTC is independent of daylight saving time and provides a unique time value for the entire world. 
+The definition of UTC started from the epoc `1970-01-01T00:00:00Z`. Initially, humans formed a day into 86400 seconds.
```
Comment (praneesha) [inline time-ballerina/Module.md:7]:
```suggestion
The definition of UTC started from the epoc `1970-01-01T00:00:00Z`. Initially, humans divided a day into 86400 seconds. 
```

File: time-ballerina/Module.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
+## Overview
+
+This module provides a set of APIs that have the capabilities to generate and manipulate UTC and localized time.
+
+In cloud computing, the most essential type of time is UTC. Coordinated Universal Time (UTC) is the primary time standard on which the world agreed. 
+UTC is independent of daylight saving time and provides a unique time value for the entire world. 
+The definition of UTC started from the epoc `1970-01-01T00:00:00Z`. Initially, humans formed a day into 86400 seconds. 
+However, the Earth rotation does not adhere to this time duration as Earth slows, and the day is getting longer.
```
Comment (praneesha) [inline time-ballerina/Module.md:8]:
```suggestion
However, the Earth rotation does not adhere to this time duration as the Earth is slowing down and the day is getting longer. 
```

File: time-ballerina/Module.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
+## Overview
+
+This module provides a set of APIs that have the capabilities to generate and manipulate UTC and localized time.
+
+In cloud computing, the most essential type of time is UTC. Coordinated Universal Time (UTC) is the primary time standard on which the world agreed. 
+UTC is independent of daylight saving time and provides a unique time value for the entire world. 
+The definition of UTC started from the epoc `1970-01-01T00:00:00Z`. Initially, humans formed a day into 86400 seconds. 
+However, the Earth rotation does not adhere to this time duration as Earth slows, and the day is getting longer. 
+As a result, a solar day in 2012 is longer than 86400 SI seconds. 
+To correct this incompatibility, additional seconds have been added to the UTC scale which is known as leap-seconds.
```
Comment (praneesha) [inline time-ballerina/Module.md:10]:
```suggestion
To correct this incompatibility, additional seconds have been added to the UTC scale, which is known as leap-seconds.
```

File: time-ballerina/Module.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+
+The more focus on this module is to give the most precise UTC with nanoseconds precision and also handle some complex use cases such as leap seconds and daylight time-saving.
```
Comment (praneesha) [inline time-ballerina/Module.md:12]:
```suggestion
The focus of this module is to give the most precise UTC with nanoseconds precision and also handle some complex use cases such as leap seconds and daylight time-saving.
```

File: time-ballerina/Module.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+### Monotonic Time
+The monotonic time represents the number of seconds from an unspecified epoch.
+
+Use the following API to get the monotonic time from an unspecified topic:
+```ballerina
+decimal seconds = time:monotonicNow();
+```
+
+### Civil Time
+The localized time represents using `time:Civil` record. It includes the following details:
```
Comment (praneesha) [inline time-ballerina/Module.md:32]:
```suggestion
The localized time represents using the `time:Civil` record. It includes the following details:
```

File: time-ballerina/Module.md [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+- timezone information
+- daylight time-saving information
+
+### APIs
+Parallel to the aforementioned time representations, this module includes a set of APIs to facilitate time conversions
+and manipulations using a set of high-level APIs. Those conversion APIs can be listed as follows.
+
+### The String Representations of UTC
+```ballerina
+// Converts from RFC 3339 timestamp to UTC
```
Comment (praneesha) [inline time-ballerina/Module.md:44]:
```suggestion
// Converts from RFC 3339 timestamp to UTC.
```

File: time-ballerina/Module.md [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+time:Utc utc = check time:utcFromString("2007-12-0310:15:30.00Z");
+
+// Converts a given time:Utc time to a RFC 3339 timestamp
```
Comment (praneesha) [inline time-ballerina/Module.md:47]:
```suggestion
// Converts a given `time:Utc` time to a RFC 3339 timestamp.
```

File: time-ballerina/Module.md [hunk 12]
```diff
@@ -0,0 +1,79 @@
...
+string utcString = time:utcToString(utc);
+```
+
+### The String Representations of Civil
+```ballerina
+// Converts from RFC 3339 timestamp to a civil record
```
Comment (praneesha) [inline time-ballerina/Module.md:53]:
```suggestion
// Converts from RFC 3339 timestamp to a civil record.
```

File: time-ballerina/Module.md [hunk 13]
```diff
@@ -0,0 +1,79 @@
...
+time:Civil civil2 = check time:civilFromString("2007-12-03T10:15:30.00Z");
+
+// Converts a given time:Civil time to a RFC 3339 timestamp
```
Comment (praneesha) [inline time-ballerina/Module.md:56]:
```suggestion
// Converts a given `time:Civil` time to a RFC 3339 timestamp.
```

File: time-ballerina/Module.md [hunk 14]
```diff
@@ -0,0 +1,79 @@
...
+string civilString = check time:civilToString(civil);
+```
+
+### UTC Value Manipulation
+```ballerina
+// Returns the UTC time that occurs seconds after the given UTC
```
Comment (praneesha) [inline time-ballerina/Module.md:62]:
```suggestion
// Returns the UTC time that occurs seconds after the given UTC.
```

File: time-ballerina/Module.md [hunk 15]
```diff
@@ -0,0 +1,79 @@
...
+time:Utc utc = time:utcAddSeconds(time:utcNow(), 20.900);
+
+// Returns difference in seconds between two given UTC time values
```
Comment (praneesha) [inline time-ballerina/Module.md:65]:
```suggestion
// Returns the difference in seconds between two given UTC time values.
```

File: time-ballerina/Module.md [hunk 16]
```diff
@@ -0,0 +1,79 @@
...
+time:Utc utc1 = time:utcNow();
+time:Utc utc2 = check time:utcFromString("2021-04-12T23:20:50.520Z");
+time:Seconds seconds = time:utcDiffSeconds(utc1, utc2);
+```
+
+### UTC vs Civil
+```ballerina
+// Converts a given UTC to a Civil
```
Comment (praneesha) [inline time-ballerina/Module.md:73]:
```suggestion
// Converts a given UTC to a Civil.
```

File: time-ballerina/Module.md [hunk 17]
```diff
@@ -0,0 +1,79 @@
...
+time:Civil civil = time:utcToCivil(utc);
+
+// Converts a given Civil to an UTC
```
Comment (praneesha) [inline time-ballerina/Module.md:76]:
```suggestion
// Converts a given Civil to a UTC.
```

File: time-ballerina/time_apis.bal [hunk 18]
```diff
@@ -15,12 +15,12 @@
 // under the License.
 import ballerina/jballerina.java;
 
-# Returns Utc representing current time(current instant of the system clock as seconds from the epoch of 1970-01-01T00:00:00).
-# + precision - Specifies number of zeros after decimal point (e.g. 3 would give millisecond precision
-# and nil means native precision(nanosecond precision 9) of clock)
+# Returns UTC representing current time(current instant of the system clock as seconds from the epoch of 1970-01-01T00:00:00).
```
Comment (praneesha) [inline time-ballerina/time_apis.bal:18]:
```suggestion
# Returns the UTC representing the current time (current instant of the system clock in seconds from the epoch of 1970-01-01T00:00:00).
```

File: time-ballerina/time_apis.bal [hunk 19]
```diff
@@ -15,12 +15,12 @@
...
 # ```ballerina
 # time:Utc utc = time:utcNow();
 # ```
+# + precision - Specifies number of zeros after decimal point (e.g. 3 would give millisecond precision
```
Comment (praneesha) [inline time-ballerina/time_apis.bal:22]:
```suggestion
# + precision - Specifies the number of zeros after the decimal point (e.g., 3 would give the millisecond precision
```

File: time-ballerina/time_apis.bal [hunk 20]
```diff
@@ -15,12 +15,12 @@
...
+# and nil means native precision(nanosecond precision 9) of clock)
```
Comment (praneesha) [inline time-ballerina/time_apis.bal:23]:
```suggestion
# and nil means native precision (nanosecond precision 9) of the clock)
```

File: time-ballerina/time_types.bal [hunk 21]
```diff
@@ -23,7 +23,7 @@ public type Seconds decimal;
 # compared using the Ballerina <, <=, >, >= operators.
 # First member of tuple is int representing integral number of
 # seconds from the epoch.
-# Epoch is traditional Unix epoch of 1970-01-01T00:00:00Z
+# Epoch is traditional UNIX epoch of 1970-01-01T00:00:00Z
```
Comment (praneesha) [inline time-ballerina/time_types.bal:26]:
```suggestion
# Epoch is the traditional UNIX epoch of 1970-01-01T00:00:00Z.
```

File: time-ballerina/time_types.bal [hunk 22]
```diff
@@ -23,7 +23,7 @@ public type Seconds decimal;
 # compared using the Ballerina <, <=, >, >= operators.
 # First member of tuple is int representing integral number of
```
Comment (praneesha) [inline time-ballerina/time_types.bal:24]:
```suggestion
# The first member of the tuple is int representing an integral number of
```

File: time-ballerina/time_types.bal [hunk 23]
```diff
@@ -23,7 +23,7 @@ public type Seconds decimal;
...
 # seconds from the epoch.
-# Epoch is traditional Unix epoch of 1970-01-01T00:00:00Z
+# Epoch is traditional UNIX epoch of 1970-01-01T00:00:00Z
 # Second member of tuple is decimal giving the fraction of
```
Comment (praneesha) [inline time-ballerina/time_types.bal:27]:
```suggestion
# The second member of the tuple is a decimal giving the fraction of
```

**Comment by BuddhiWathsala** [comment]
@praneesha, the suggestions are fixed in here https://github.com/ballerina-platform/module-ballerina-time/pull/139/commits/ca01f551cf1aa6ffcbc1b46dea6500f35ed21895

File: time-ballerina/Package.md [hunk 24]
```diff
@@ -1,73 +1,16 @@
 ## Package Overview
 
-This package provides implementations related to the UTC and localized time. In the world of cloud computing, the most essential type of time is UTC. 
-Therefore, the more focus on this module is to give the most precise UTC and also handle some complex use cases such as leap seconds and daylight time-saving.
+The Time library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina
```
Comment (praneesha) [inline time-ballerina/Package.md:3]:
```suggestion
The `time` library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina
```

**Review by praneesha — APPROVED** [review]

**Pull request merged**